mozilla-mobile/firefox-ios
# Add FXIOS-8174 [v124] Data clearance animation

## :scroll: Tickets
[Jira ticket](https://mozilla-hub.atlassian.net/browse/FXIOS-8174)
[Github issue](https://github.com/mozilla-mobile/firefox-ios/issues/18184)

## :bulb: Description
Add animation to data clearance flow using Lottie. 
* Use combined animation file for iPhone
* Separate animation file for iPad to use gradient and wave so that the gradient can conform to different layouts without modifying the wave view. 

**Testing Steps**
See screenshots below to confirm animation for iPhone and iPad
Verify animations look good in split view
Ensure animation does not appear when Reduce Motion setting is enabled in Accessibility

## :pencil: Checklist
You have to check all boxes before merging

# Screenshots

https://github.com/mozilla-mobile/firefox-ios/assets/6743397/2f770a36-eae0-4cba-90df-8c607f9ecc12


https://github.com/mozilla-mobile/firefox-ios/assets/6743397/4fd7005d-15dd-4b6c-a24d-01a1607f68af



https://github.com/mozilla-mobile/firefox-ios/assets/6743397/716ff01b-a23b-49e1-8697-f76a96cc7c76
